qrust-api/src/main/resources/application-harvey.yml

```
@@ -18,4 +18,10 @@ spring:
 
 logging.level:
   org.hibernate.SQL: debug # 이거 끄면 format_sql 적용 안됨
-#  org.hibernate.type: trace
\ No newline at end of file
+#  org.hibernate.type: trace
+
+minio:
+  endpoint: http://mayfifth99.store:9000
+  access-key: minioadmin
+  secret-key: minioadmin
+  bucketName: qrust-bucket
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)